backend/database_handler/validators_registry.py

```diff
@@ -25,14 +40,17 @@ def _parse_validator_data(self, validator_data: dict) -> dict:
 
     def _get_validator_or_fail(self, validator_address: str):
         """Private method to check if an account exists, and raise an error if not."""
-        condition = f"address = '{validator_address}'"
-        result = self.db_client.get(self.db_validators_table, condition)
 
-        validator_data = result[0] if result else None
+        with Session(self.db_client.engine) as session:
+            validator_data = (
+                session.query(models.Validators)
+                .filter_by(address=validator_address)
+                .first()
+            )
 
         if not validator_data:
             raise ValidatorNotFound(validator_address)
-        return self._parse_validator_data(validator_data)
+        return validator_data.to_dict()
```

Comment:
@AgustinRamiroDiaz, is this to_dict method the one defined in the models? If so, the one defined in this file is not currently used. Are we keeping it for the future?

Reply:
@cristiam86 you are right! Fixed